service_management/azure/azure.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('systemu',                 '~> 2.6')
   s.add_runtime_dependency('thor',                    '~> 0.19')
   s.add_runtime_dependency('uuid',                    '~> 2.0')
+  s.add_runtime_dependency('azure-core',              '~> 0.1')
```

Comment:
There are few files says file mode changed? Just curious do you know what's that?